packages/slate-react/src/components/editable.tsx

```diff
@@ -106,7 +107,7 @@ export const Editable = (props: EditableProps) => {
     readOnly = false,
     renderElement,
     renderLeaf,
-    renderPlaceholder = props => <DefaultPlaceholder {...props} />,
```

Comment:
@ulion is there a reason for the removal of this prop?  It looks like it is used on line 1125.